src/kernel_config_manager.sh

```diff
@@ -140,6 +140,7 @@ function save_config_file()
   if [[ "$ret" == 1 ]]; then
     warning "Warning: $name: there's nothing new in this file"
   elif [[ "$ret" -gt 0 ]]; then
+    git reset --hard > /dev/null 2>&1 # Aborts save operation
```

Comment:
Do we need `/dev/null`? Can we just add `--quiet`?

Reply:
You are right 👍 
Doing it!